client/src/cmddata.c

```diff
@@ -1707,7 +1708,7 @@ int CmdHpf(const char *Cmd) {
     CLIExecWithReturn(ctx, Cmd, argtable, true);
     CLIParserFree(ctx);
 
-    uint8_t bits[g_GraphTraceLen];
+    uint8_t *bits = malloc(g_GraphTraceLen);
```

Comment:
reminder: **_all_** calls to malloc / calloc need to be checked for successful allocation.

Reply:
I see. I've added the check to all of the memory allocation code in commit 8b6a274e289cd96f16616de741f0d02fb10d1e6e .
I did ignored the check when writing these code. Thanks!